src/helpers/blocks-helper.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * A helper object for blocks.
+ *
+ * @package Yoast\WP\Free\Helpers
+ */
+
+namespace Yoast\WP\Free\Helpers;
+
+use WP_Block_Parser_Block;
+
+class Blocks_Helper {
+
+	/**
+	 * @param $post_id
+	 *
+	 * @return array
+	 */
+	public function get_all_blocks_from_post( $post_id ) {
+		if ( ! \function_exists( 'parse_blocks' ) ) {
```

Comment:
This method is calling `get_all_blocks_from_content ` which performs the same check. I would remove this check here and rely on the one in `get_all_blocks_from_content`.

Reply:
That would mean potentially loading the post before doing the check. Considering the check is cheap whereas loading the post may not be ( depending on whether or not it's already been loaded ) I'm partial to leaving it as it currently is.